clients/src/test/java/org/apache/kafka/common/metrics/MetricsTest.java

```diff
@@ -475,28 +464,13 @@ public void testPercentiles() {
         Metric p50 = this.metrics.metrics().get(metrics.metricName("test.p50", "grp1"));
         Metric p75 = this.metrics.metrics().get(metrics.metricName("test.p75", "grp1"));
 
-        // record two windows worth of sequential values
-        for (int i = 0; i < buckets; i++)
+        // record 1-100 sequential values
+        for (int i = 0; i < buckets; i++) {
```

Comment:
In some other places in this class, we don't use {} for single line statements. Should we be consistent?

Reply:
Applied